### DISCUSSION TITLE
connecting ApicurioSudio to a remote keycloakserver

### DISCUSSION THREAD

**Discussion by ivanbishop** [ApicurioArchive/apicurio-studio#2138] [Q&A]
I have a keycloak server running on an AWS EC2 instance on (say) 10.1.1.10:8443 with a realm (called apicurio) imported  from https://github.com/Apicurio/apicurio-studio/tree/master/distro/docker-compose/config/keycloak

I see the realm called "apicurio" in the keycloak UI.

I need some advice on configuring two items 

[1]  In the Apicurio Realm on Keycloak server 10.1.1.10
      RootURL: I have https://10.1.1.10:8443/
         and 
       Valid redirect as  https://10.1.1.10:8443/*

AND

[2]  On the Apicurio Studio server 10.1.1.100 (say) what do I set the properties to for these in standalone-apicurio.xml?

 property name="apicurio.kc.auth.rootUrl" value="https://10.132.17.243:8443"
 property name="apicurio.kc.auth.realm" value="apicurio"

When I browse 10.1.1.100:8080 I get redirected to https://studio-auth.apicur.io/auth/realms/apicurio-local/protocol

I see no packets going to keycloak on 10.1.1.10 (I'm tcpdumping the packets)

How do I make Studio on 10.1.1.100 go to my apicurio realm on the keycloak server10.1.1.10 ?

Thanks

---EXTRA INFO---

**OPEN PORTS on Apicurio Server**

#netstat -ntlp
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp        0      0 0.0.0.0:8443            0.0.0.0:*               LISTEN      9226/java
tcp        0      0 127.0.0.53:53           0.0.0.0:*               LISTEN      421/systemd-resolve
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      784/sshd: /usr/sbin
tcp        0      0 127.0.0.1:6010          0.0.0.0:*               LISTEN      10450/sshd: ubuntu@
tcp        0      0 127.0.0.1:9990          0.0.0.0:*               LISTEN      9226/java
tcp        0      0 0.0.0.0:8080            0.0.0.0:*               LISTEN      9226/java
tcp6       0      0 ::1:6010                :::*                    LISTEN      10450/sshd: ubuntu@
tcp6       0      0 :::22                   :::*                    LISTEN      784/sshd: /usr/sbin

**P.S.> this challenge seems related to this OLD chat from 2019**
https://github.com/Apicurio/apicurio-studio/issues/926

**Comment by EricWittmann** [comment]
Sorry I never responded to this.  I wonder if this was ever resolved.  :(  It should be possible, but the version of Studio in question is now quite old and not really supported anymore.  Apologies.
